src/Microsoft.Extensions.Logging.Filter/IFilterLoggerSettings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <summary>
+    /// Filter settings for messages logged by an <see cref="ILogger"/>.
+    /// </summary>
+    public interface IFilterLoggerSettings
```

Comment:
Does the `IFilterLoggerSettings` interface really need the `ChangeToken` and `Reload` functionality? They are not used within this library. (`FilterLoggerSettings` only has a dummy implementation) 

Shouldn't this be part of implementations that actually support it - e.g. a `Configuration`-based implementation?